test/python/test_optional_dependency.py

```
@@ -0,0 +1,178 @@
+#
+# SPDX-License-Identifier: MIT
+#
+# Licensed under the MIT License
+
+"""Tests for OptionalDependencyTester."""
+
+from __future__ import annotations
+
+import pytest
+
+from mqt.core._compat import OptionalDependencyTester  # noqa: PLC2701
```

Ruff (0.14.3)

Unused `noqa` directive (non-enabled: `PLC2701`)

Remove unused `noqa` directive

(RUF100)